docs/source/ucx.rst

```diff
@@ -1,135 +1,199 @@
+  Replaces ``sockcm`` with ``rdmacm`` in ``UCX_TLS`` and ``UCX_SOCKADDR_TLS_PRIORITY``, enabling remote direct memory access (RDMA) for InfiniBand transfers.
+  This is recommended by UCX for use with InfiniBand, and will not work it is disabled.
 
-.. code-block:: bash
+- ``ucx.net-devices: <str>`` -- **recommended.**
+
+  Explicitly sets ``UCX_NET_DEVICES`` instead of defaulting to ``"all"``, which can result in suboptimal performance.
+  If using InfiniBand, set to ``"auto"`` to automatically detect the InfiniBand interface closest to each GPU.
+  If InfiniBand is disabled, set to a UCX-compatible ethernet interface, e.g. ``"enp1s0f0"`` on a DGX-1.
+  All available UCX-compatible interfaces can be listed by running ``ucx_info -d``.
+
+  .. warning::
+      Setting ``ucx.net-devices: "auto"`` assumes that all InfiniBand interfaces on the system are connected and properly configured; undefined behavior may occur otherwise.
+  
+
+- ``rmm.pool-size: <str|int>`` -- **recommended.**
+
+  Allocates an RMM pool of the specified size for the process; size can be provided with an integer number of bytes or in human readable format, e.g. ``"4GB"``.
+  It is recommended to set the pool size to at least the minimum amount of memory used by the process; if possible, one can map all GPU memory to a single pool, to be utilized for the lifetime of the process.
+
+.. note::
+    These options can be used with mainline Dask/Distributed.
+    However, this will disable a variety of features, such as the automatic detection of InfiniBand interfaces. 
+    See :doc:`Specializations for GPU Usage <specializations>` for more details on the benefits of using Dask-CUDA.
 
-    DASK_RMM__POOL_SIZE=1GB DASK_UCX__CUDA_COPY=True DASK_UCX__TCP=True DASK_UCX__NVLINK=True dask-scheduler --protocol ucx --interface eth0
 
-Setting ``DASK_UCX__NET_DEVICES`` when using an interface that isn't an InfiniBand can generally be skipped.
+Usage
+-----
 
+Dask-CUDA workers using UCX communication can be started manually with the ``dask-cuda-worker`` CLI tool or automatically with ``LocalCUDACluster``.
+In either case, a ``distributed.Client`` must be made for the worker cluster using the same UCX configuration.
 
 dask-cuda-worker
 ^^^^^^^^^^^^^^^^
 
-All ``DASK_*`` configurations described above have analogous parameters in ``dask-cuda-worker`` which are preferred over the regular configurations used for ``dask-scheduler`` due to some specializations, such as ``--net-devices="auto"`` which will correctly assign the topologically closest IB interface to the GPU of each worker, something that's not possible with ``DASK_UCX__NET_DEVICES``.
+A Dask cluster with UCX support can be started using the ``dask-cuda-worker`` CLI tool with a Dask scheduler which has been configured for UCX.
+
+*Do we want to include any reasons you would do this instead of using LocalCUDACluster?*
```

Comment:
`LocalCUDACluster` is single-node only, anything else requires `dask-cuda-worker`.